airesources/Scala/run_game.bat

```diff
@@ -1,2 +1,3 @@
-call scalac *.scala hlt\*.scala
-halite -d "240 160" "scala MyBot" "scala MyBot"
+sbt assembly
+\halite.exe -d "240 160" "java -jar target\scala-2.12\MyBot.jar" "java -jar target\scala-2.12\MyBot.jar"
```

Comment:
Remove \ in front of halite - it's not going to run like that.

Reply:
Thanks, haven't used windows for a while.